src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -616,3 +616,8 @@
 	z-index: 33;
 	background-color: var(--vscode-terminal-background, --vscode-panel-background);
 }
+
+.monaco-workbench .accessible-buffer {
+	pointer-events: none;
```

Comment:
Is this going to keep me from selecting/copying though?

Reply:
Seems like it should be `display: none` when not active

Reply:
yeah when it's not active, you shouldn't be able to select or copy. see the css rule above that will allow you to do all of that

Reply:
Ah I get it